### ISSUE TITLE
Should the touch input option turn hasPointerEvents off?

### ISSUE THREAD

**Issue by aNNiMON** [nikita36078/J2ME-Loader#173]
Should the new option 'Touch input' turn off `hasPointerEvents()` and `hasPointerMotionEvents()` in Canvas? Now both methods [rely on overlay nullability](https://github.com/nikita36078/J2ME-Loader/blob/8d08f541f39be5078ca15fa5603349074f7fbace/app/src/main/java/javax/microedition/lcdui/Canvas.java#L530-L536). 
For example, it can be useful in Gravity Defied to disable annoying circle control
![screen](https://user-images.githubusercontent.com/1321838/36858984-bccfb590-1d85-11e8-9c1c-b202114bd1a6.png)

**Comment by nikita36078** [comment]
Yes, hasPointerEvents() and hasPointerMotionEvents() seem to be unreliable now, it should fix them.

**Comment by nikita36078** [comment]
Done. efb0ea08c6bcb7343eceebf910b5cb715578b3f0
